tests/rptest/services/redpanda_installer.py

```diff
@@ -26,7 +26,7 @@
 VERSION_RE = re.compile(".*v(\\d+)\\.(\\d+)\\.(\\d+).*")
 # strict variant of VERSION_RE that only matches "vX.Y.Z" strings
 STRICT_VERSION_RE = re.compile(r"^v(\d+)\.(\d+)\.(\d+)$")
-RELEASES_CACHE_FILE = "/tmp/redpanda_releases.json"
+RELEASES_CACHE_FILE = "/tmp/ducktape_cache/redpanda_releases.json"
```

Comment:
>`ducktape_cache`

this is new for this PR right?

Reply:
Yes, I'll merge this in and rerun CI on the pair PR to run the tests with this being cached as well.